main.go

```diff
@@ -135,7 +135,7 @@ func mainE(ctx context.Context) error {
 	daemonCommand.PersistentFlags().String(f.Service.Prometheus.Retention.Duration, "2w", "Retention duration for prometheus.")
 	daemonCommand.PersistentFlags().String(f.Service.Prometheus.Retention.Size, "90Gi", "Retention size for prometheus.")
 	daemonCommand.PersistentFlags().String(f.Service.Prometheus.Storage.Size, "100Gi", "Storage size for prometheus.")
-	daemonCommand.PersistentFlags().String(f.Service.Prometheus.Version, "v2.39.1", "Prometheus container image version.")
+	daemonCommand.PersistentFlags().String(f.Service.Prometheus.Version, "v2.41.0", "Prometheus container image version.")
```

Comment:
Why 2.41.0 when changelog says 2.40.1 ?
BTW, 2.41.0 is only RC for the moment, latest stable is 2.40.7.

Reply:
Becaue of this https://github.com/prometheus-community/helm-charts/blob/f4555244480525b46a6fad320960955d655ca376/charts/kube-prometheus-stack/values.yaml#L2666

Reply:
I updated the changelog

Reply:
And actually it's not RC now :D https://github.com/prometheus/prometheus/releases/tag/v2.41.0

Reply:
erf, changelog on their master branch is not up to date :disappointed: